build.sbt

```diff
@@ -4255,7 +4256,7 @@ lazy val `os-environment` =
         val targetDir = (Test / target).value
         NativeImage.buildNativeImage(
           "test-os-env",
-          staticOnLinux = true,
+          staticOnLinux = false,
```

Comment:
Why is this change needed?

Reply:
- one of the tests now uses `dl_open` to read `libjni.so`
- dynamic loading isn't support with MUSL